source/extensions/filters/http/fault/fault_filter.cc

```diff
@@ -51,7 +51,8 @@ FaultSettings::FaultSettings(const envoy::extensions::filters::http::fault::v3::
           fault, max_active_faults_runtime, RuntimeKeys::get().MaxActiveFaultsKey)),
       response_rate_limit_percent_runtime_(
           PROTOBUF_GET_STRING_OR_DEFAULT(fault, response_rate_limit_percent_runtime,
-                                         RuntimeKeys::get().ResponseRateLimitPercentKey)) {
+                                         RuntimeKeys::get().ResponseRateLimitPercentKey)),
+      disable_downstream_server_tracing_{fault.disable_downstream_server_tracing()} {
```

Comment:
That's the first time I've seen using curly brackets in MIL.
It seems to be doing the same thing as parentheses in this case, but usually I see curly brackets used for non member initialization.

Reply:
Thanks, will fix.